test/regress_http.c

```diff
@@ -4309,6 +4309,102 @@ static void http_simple_test(void *arg)
 static void http_simple_nonconformant_test(void *arg)
 { http_simple_test_impl(arg, 0, 0, "/test nonconformant"); }
 
+static const char *
+http_get_addr(int mask)
+{
+	if (mask & HTTP_BIND_IPV6)
+		return "::1";
+	else
+		return "127.0.0.1";
+}
+static int
+https_bind_ssl_bevcb(struct evhttp *http, ev_uint16_t port, ev_uint16_t *pport, int mask)
+{
+	int _port;
+	struct evhttp_bound_socket *sock = NULL;
+	sock = evhttp_bind_socket_with_handle(http, http_get_addr(mask), port);
+
+#ifdef EVENT__HAVE_OPENSSL
+	if (mask & HTTP_OPENSSL) {
+		init_ssl();
```

Comment:
I'm not sure that this is OK to call it multiple times, it is definitely safe for openssl 1.1 - since it is no-op, but not sure about older versions